src/node/test/historical_queries.cpp

```diff
@@ -462,21 +881,19 @@ TEST_CASE("Recover historical ledger secrets")
     // Provide target and subsequent entries until next signature
     for (size_t i = second_index; i <= signature_index; ++i)
     {
-      REQUIRE(provide_ledger_entry(i));
+      provide_ledger_entry(i);
```

Comment:
Several of these `provide_ledger_entries` can no longer be `REQUIRE`d because the `StateCache` may already have this entry from a previous request, and not have asked for it again. We could be precise about which were asked for (the return value of this should always match "did the Cache ask for it"), but for simplicity for now I've just dropped checking some of these return values.